pkg/util/constants/constants_nix.go

```diff
@@ -0,0 +1,9 @@
+//go:build netbsd || openbsd || solaris || dragonfly || linux
+// +build netbsd openbsd solaris dragonfly linux
+
+package constants
+
+const (
+	//DefaultDogstatsDLogFile points to the dogstatsd stats log file that will be used if not configured
+	DefaultDogstatsDLogFile = "/var/log/datadog/dogstatsd_info/dogstatsd-stats.log"
```

Comment:
I support the usage of this new `constants` pkg.
Can we move the constants from the existing `cmd/agent/common` pkg into this too?

Reply:
I moved all cons and vars from `common` pkg to `constants` pkg. Also change all other codes that are using the cons from `common` to `constants` pkg accordingly